src/main/java/address/shortcuts/Bindings.java

```diff
@@ -0,0 +1,161 @@
+package address.shortcuts;
+
+import address.events.*;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCodeCombination;
+import javafx.scene.input.KeyCombination;
+import javafx.scene.input.KeyEvent;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Contains the mapping of shortcuts and the corresponding event to raise
+ */
+public class Bindings {
+
+    /**
+     * List of accelerators used.
+     * They are here for the purpose of record keeping. Handled automatically by JavaFX.
+     */
+    private List<Accelerator> accelerators = new ArrayList<>();
+
+    /**
+     * List of Global hotkeys.
+     */
+    private List<GlobalHotkey> hotkeys = new ArrayList<>();
+
+    /**
+     * List of keyboard shortcuts.
+     */
+    private List<Shortcut> shortcuts = new ArrayList<>();
+
+
+    /* Global hotkeys in alphabetical order of name */
+    public List<GlobalHotkey> HOTKEY_APP_MINIMIZE = new ArrayList<>();
+    public List<GlobalHotkey> HOTKEY_APP_MAXIMIZE = new ArrayList<>();
```

Comment:
*Minor -* the comment mentions alphabetical order, but that means `MAXIMIZE` should come before `MINIMIZE`